pkg/policy/k8s/cilium_cidr_group.go

```diff
@@ -231,20 +235,32 @@ func translateSpec(spec *api.Rule, cidrsSets map[string][]api.CIDR) {
 		spec.Ingress[i].FromCIDRSet = cidrSet
 	}
 	for i := range spec.IngressDeny {
+		if spec.IngressDeny[i].FromCIDRSet == nil {
+			continue
+		}
+
 		cidrSet := translateCIDRRuleSlice(spec.IngressDeny[i].FromCIDRSet, cidrsSets)
 		if cidrSet == nil {
 			cidrSet = make([]api.CIDRRule, 0)
 		}
 		spec.IngressDeny[i].FromCIDRSet = cidrSet
 	}
 	for i := range spec.Egress {
+		if spec.Egress[i].ToCIDRSet == nil {
+			continue
+		}
+
 		cidrSet := translateCIDRRuleSlice(spec.Egress[i].ToCIDRSet, cidrsSets)
 		if cidrSet == nil {
 			cidrSet = make([]api.CIDRRule, 0)
 		}
 		spec.Egress[i].ToCIDRSet = cidrSet
 	}
 	for i := range spec.EgressDeny {
+		if spec.EgressDeny[i].ToCIDRSet == nil {
+			continue
+		}
+
 		cidrSet := translateCIDRRuleSlice(spec.EgressDeny[i].ToCIDRSet, cidrsSets)
```

Comment:
The code below seems to want to make sure that the `ToCIDRSet` isn't `nil`, but now we're specifically letting it be `nil` - does code downstream handle this well? I remember a bug around this where we had nil pointer panics :sweat_smile: